app/components/layouts/header.scss

```diff
@@ -140,38 +140,39 @@
   cursor: pointer;
 }
 
-.signOutButton {
+.profileButton {
+  width: 155px !important;
+  height: 44px !important;
+  font-size: 14px !important;
   position: relative;
-  text-align: center !important;
   z-index: 999999;
+  padding: 0 !important;
+
+  &:hover {
+    font-weight: bold;
+    background-color: $gray0 !important;
+  }
+
+  a {
```

Comment:
I think of that it's unsafe and anti-pattern to apply CSS style with the element name.
If there isn't any extra reason to use this method, please use the class-based selector.